usim.py

```diff
@@ -384,7 +384,8 @@ def monitor_sub(q, index):
 	t0 = time.time()
 	log('Started main thread %d' % (index+1))
 	while True:
-		log("Restarting loop", console_only=True)
+		#log("Restarting loop", console_only=True)
```

Comment:
Sometimes its useful to just comment out debugging code (like this appears to be) so that when debugging you can just uncomment it. Its like a lazy persons logging directive imo, so it makes sense